### ISSUE TITLE
Metal3 - Network interfaces - Use specific naming scheme

### ISSUE THREAD

**Issue by rletrocquer** [suse-edge/charts#153]
Since metal3:0:8:0 and the use of a ramdisk image based on Leap/SLES 15.6, the names of the network interfaces detected during introspection have changed
As explained by @hardys `metal3 0.7.x` is 15.5 based ramdisk which used `systemd-249.17` while the `metal3 0.8.x` ramdisk is 15.6 based and used `systemd-254.15`

metal3:0:7:x :
```
network_interfaces:
      bond0:
        type: bond
        interfaces:
          - enp75s0f0
          - enp75s0f1
```

metal3:0.8.x :
```
network_interfaces:
      bond0:
        type: bond
        interfaces:
          - enp75s0f0np0
          - enp75s0f1np1
```

We have tried to define `net.naming-scheme=v249 ` in IronicKernelParams without success for the moment.

**Comment by hardys** [comment]
Sorry for the slow response @rletrocquer - I looked into this and I believe this may be related to a device driver change, specifically I think the Intel i40e driver gained devlink support, which impacts the nic naming.

Unfortunately that naming isn't controlled by the systemd naming-scheme, and even if we can configure the driver to enable the old naming, it will likely then mean the IPA names are inconsistent with newer kernels.

I'm open to suggestions re how we proceed here, but currently I think this is unfortunately a migration path issue related to kernel updates, and it will be impossible to ensure the IPA names always align with different older/newer target OS kernels.
